test/transforms.jl

```diff
@@ -41,4 +41,24 @@
     @test nelements(smesh) == nelements(mesh)
     @test topology(smesh) == topology(mesh)
   end
+
+  @testset "ScaleCoords" begin
+    # a scaling doesn't change the topology
```

Comment:
Please add a test at the beginning testing that both constructors work, i.e. ScaleCoords([1,2,3]) == ScaleCoords(1,2,3)